app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/auth/AuthNavigation.kt

```diff
@@ -189,6 +190,23 @@ fun NavGraphBuilder.authGraph(
                 navController.popUpToCompleteRegistration()
             },
         )
+        expiredRegistrationLinkDestination(
+            onNavigateBack = { navController.popBackStack() },
+            onNavigateToStartRegistration = {
+                navController.navigateToStartRegistration(
+                    navOptions = navOptions {
+                        popUpTo(LANDING_ROUTE)
+                    },
+                )
+            },
+            onNavigateToLogin = {
+                navController.navigateToLanding(
+                    navOptions = navOptions {
+                        popUpTo(LANDING_ROUTE)
```

Comment:
This doesn't put a second LandingScreen on the stack, right?

Reply:
no but you beat me to pushing a change to address some funkiness I messed up with the root nav screen and rootNavOptions